api/users/urls.py

```diff
@@ -6,6 +6,6 @@
     url(r'^$', views.UserList.as_view(), name=views.UserList.view_name),
     url(r'^(?P<user_id>\w+)/$', views.UserDetail.as_view(), name=views.UserDetail.view_name),
     url(r'^(?P<user_id>\w+)/nodes/$', views.UserNodes.as_view(), name=views.UserNodes.view_name),
-    url(r'^(?P<user_id>\w+)/registrations/$', views.UserRegistrations.as_view(), name=views.UserRegistrations.view_name)
-
+    url(r'^(?P<user_id>\w+)/node_logs/$', views.UserNodeLogs.as_view(), name=views.UserNodeLogs.view_name),
```

Comment:
Well  `users/me/nodes` are nodes that the user has contributed to, not just nodes the user created. But in the documentation, we clarify, and say the endpoint is a "Paginated list of nodes that the user contributes to". 

In my opinion, calling them `node_logs` still doesn't clear up the confusion as to who created them, and only complicates things by having the user think that registration logs are excluded. They are not.